src/openforms/config/models.py

```diff
@@ -587,6 +587,16 @@ class GlobalConfiguration(SingletonModel):
         validators=[MaxValueValidator(60)],
     )
 
+    recipients_email_digest = ArrayField(
+        models.EmailField(),
+        verbose_name=_("recipients email digest"),
+        help_text=_(
+            "The email addresses that should receive a daily report of any email sending failures."
```

Comment:
```suggestion
            "The email addresses that should receive a daily report of items requiring attention."
```
while we're starting out with email sending failures, I reckon this mechanism can be used much broader and be used to inform an admin on any issues that need investigating.